### DISCUSSION TITLE
Renovate is automerging with failing CI

### DISCUSSION THREAD

**Discussion by luke-hill** [renovatebot/renovate#24179] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Check here. Why did this automerge? It has 1 failing CI run

CI run here (This has happened several times):
- https://github.com/cucumber/cucumber-ruby/pull/1724
- https://github.com/cucumber/cucumber-ruby/pull/1718
- more ....

Config here: https://github.com/cucumber/renovate-config/blob/main/default.json

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by viceice** [comment]
You need required status checks or disable renovate platform automerge

**Reply by luke-hill** [reply]
But this doesn't make sense. Because the auto-merge docs stipulate it will automerge based on CI passing. Unless I'm reading the wrong docs?

The very first line here: https://docs.renovatebot.com/key-concepts/automerge/#introduction

**Reply by viceice** [reply]
only if you don't enable github automerge. since renovate v36 we enabled platform automerge, so renovate is using the github automerge when available.
github only waits for mandatory status checks before merge.

**Reply by luke-hill** [reply]
I feel as though we're talking about diff things. All 15 items are mandatory in the CI pipeline? I am happy to concede maybe something is configured wrong. But it's not evident.

**Reply by luke-hill** [reply]
Think I've managed to spot the issue, but it wasn't self evident. Will close this and re-open if issue continues
